automated install/basic-install.sh

```diff
@@ -668,9 +668,10 @@ install_dependent_packages() {
 	declare -a argArray1=("${!1}")
 
 	for i in "${argArray1[@]}"; do
-		echo -n ":::    Checking for $i..."
-		package_check_install "${i}" &> /dev/null
-		echo " installed!"
+		echo -n ":::    Installing $i..."
+		PKG_MGR_OUT=$(package_check_install "${i}" 2>&1 ) && echo "Installed!" || ( 
```

Comment:
My bash is bad, will this variable be transitory and only live for the duration of the loop? Or does it need to be `local` to the function?